activesupport/lib/active_support/duration.rb

```diff
@@ -86,7 +93,7 @@ def eql?(other)
       Duration === other && other.value.eql?(value)
     end
 
-    def hash
+    def hash # :nodoc:
```

Comment:
hmm, i think no. What you think if we say(and remove nodoc) that the method is the same as `Object#hash`?